raisinbread/RB_files/RB_Config.sql

```diff
@@ -91,5 +91,7 @@ INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (94,99,'true');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (95,49,'1000');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (96,100,'environment');
 INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (97,70,'/data-raisinbread/');
+INSERT INTO `Config` (`ID`, `ConfigID`, `Value`) VALUES (98,65,'true');
```

Comment:
What config is this for?

Reply:
I think this was generated by the Raisinbread export script so I'm not sure exactly

Reply:
Oh I think this might be for usePwnedPasswords actually. Looks like Raisinbread is missing it.

Reply:
Not sure. I'll delete it.